readme.md

```diff
@@ -102,7 +102,7 @@ Show the `Look Up [selection]` menu item when right-clicking text on macOS.
 
 #### labels
 
-Type: `Object`<br>
+Type: `Object | Function`<br>
```

Comment:
I think some more detailed information in the readme on how this is intended to be used would be helpful.

Reply:
updated